flask_admin/model/widgets.py

```diff
@@ -109,7 +109,10 @@ def get_kwargs(self, subfield, kwargs):
         elif subfield.type == 'BooleanField':
             kwargs['data-type'] = 'select'
             # data-source = dropdown options
-            kwargs['data-source'] = {'': 'False', '1': 'True'}
+            kwargs['data-source'] = json.dumps([
+                {'value': '0', 'text': gettext('No')},
```

Comment:
The `lu'string'` is not JSON serializable, so the `json.dumps` crashes if I use the `lazy_gettext`, so I have to evaluate the lazy string before serialization.